jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/forumAdministration.js

```diff
@@ -538,7 +538,7 @@ function sendForumConfiguration(e) {
                 } else {
                     jDialog.createDialog({
                         type: jDialog.alertType,
-                        bodyMessage: resp.result
+                        bodyMessage: $labelAccessDenied
```

Comment:
Don't you want to check for the error code? Currently if any error happens, you'd show access denied.